cob_light/LightMode mode
---
int32 error_type
string error_msg
